src/System/Terminal/Widgets/Common.hs

```diff
@@ -25,6 +25,7 @@ class (HasCursor w) => Widget w where
       where
         r :: w -> (Cursor, Text)
         r w = (w.cursor, toText w)
+    {-# MINIMAL handleEvent #-}
```

Comment:
What does this do?

Reply:
It makes it explicit that the only required method is `handleEvent` for instances of `Widget`. The haddock will include this entry. The other methods have defaults, meaning they aren't required. GHC can infer this, but its best to make it explicit imo.